pkg/standalone/standalone.go

```diff
@@ -1047,10 +1049,27 @@ func downloadFile(dir string, url string) (string, error) {
 		return "", nil
 	}
 
-	resp, err := http.Get(url)
+	client := http.Client{
+		Timeout: 0,
```

Comment:
@hueifeng Why the `Timeout` here is set to 0. If this is required then should not we have some +ve value?